profile/README.md

```diff
@@ -1,4 +1,10 @@
-<p align="left"><img src="https://github.com/user-attachments/assets/b31e430e-0ab6-42e3-ba43-d96f5aa61d94" width="500" /></p>
+<p align="left">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://github.com/user-attachments/assets/b31e430e-0ab6-42e3-ba43-d96f5aa61d94">
+    <source media="(prefers-color-scheme: light)" srcset="https://github.com/user-attachments/assets/c65c40eb-d39c-4859-a087-654c3ee7772c">
+    <img src="https://github.com/user-attachments/assets/b31e430e-0ab6-42e3-ba43-d96f5aa61d94" width="500" alt="GlossGenius" />
```

Comment:
question (non-blocking): Should this be `alt="GlossGenius"` or `alt="GlossGenius Logo"`

Reply:
The logo reads GlossGenius, so the reader should also read "GlossGenius" rather than saying this is the GlossGenius logo.